build.gradle

```diff
@@ -9,7 +9,7 @@ plugins {
 }
 
 group 'com.firebolt'
-version = '2.4.3'
```

Comment:
not sure whether i should create `2.4.4-SNAPSHOT` or `2.4.4`?

Reply:
Snapshot is generally pre-release, for changes that you want to test but not necessarily allow users to update to unless specifically instructed to. Non-snapshot version here should be fine IMO.